content/sensu-go/5.17/guides/secrets-management.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "Use secrets management in Sensu"
+linkTitle: "Use Secrets Management"
+description: "Sensu's secrets management allows you to avoid exposing secrets in your Sensu configuration. In this guide, you'll learn how to use Env or HashiCorp Vault as your external secrets management provider and refer to external secrets in your Sensu configuration."
```

Comment:
```suggestion
description: "Sensu's secrets management allows you to avoid exposing secrets like usernames and passwords in your Sensu configuration. In this guide, you'll learn how to use Sensu's built-in secrets provider or HashiCorp Vault and refer to external secrets in your Sensu configuration."
```